app/javascript/components/admin/UserModal.js

```diff
@@ -179,7 +160,7 @@ class UserModal extends React.Component {
                 </span>
                 {!this.state.isLocked && (
                   <ReactTooltip id="disabled-status-select" multiline={true} type="dark" effect="solid" place="bottom" className="tooltip-container">
-                    <div>{this.getTooltipText()}</div>
+                    <div>{this.state.activeState === 'Active' ? activeStatusTooltipText : inactiveStatusTooltipText}</div>
```

Comment:
since the tooltips are short, you can just put these inline

Reply:
[2353881673c67d9661dda9d52dded9830dfa8d57](https://github.com/SaraAlert/SaraAlert/pull/1180/commits/2353881673c67d9661dda9d52dded9830dfa8d57)